server/datastore/mysql/mysql.go

```diff
@@ -107,10 +107,10 @@ func (ds *Datastore) loadOrPrepareStmt(ctx context.Context, query string) *sqlx.
 	return stmt
 }
 
-// NewMDMAppleSCEPDepot returns a *apple_mdm.MySQLDepot that uses the Datastore
+// NewMDMAppleSCEPDepot returns a *mysql.SCEPDepot that uses the Datastore
```

Comment:
Tiny nit, but it returns the `scep_depot.Depot` interface instead of the actual struct pointer (just so the comment doesn't cause confusion - or maybe you meant to actually return the type and not the interface).